templates/k8s-variables.yaml

```diff
@@ -0,0 +1,81 @@
+template_dest: /tmp/k8s-ansible-service-broker.yaml
+
+dockerhub_user_name: changeme
+dockerhub_user_password: changeme
+dockerhub_org: ansibleplaybookbundle
+
+asb_src_dir: /tmp/ansible-service-broker
+
+broker_image: "ansibleplaybookbundle/ansible-service-broker:latest"
+etcd_image: "quay.io/coreos/etcd:latest"
+
+#
+# Template defaults
+#
+
+# Path of the etcd binary
+etcd_path: /usr/local/bin/etcd
+
+# Configuration filepath for Ansible Service Broker
+broker_config: /etc/ansible-service-broker/config.yaml
+
+# SSL enabled: insecure="false", termination=Reencrypt
+# Insecure enabled: insecure="true", termination=edge
+# Run in insecure mode
+insecure: "true"
```

Comment:
I thought that you said above that SSL was a limitation of the k8s? should we either comment or remove the variables that we don't need for k8s?

Reply:
This var gets ignored. I'll add a comment since I would like to add it in the future.